db/migrate/20170221003240_create_events.rb

```diff
@@ -0,0 +1,11 @@
+class CreateEvents < ActiveRecord::Migration[5.0]
+  def change
+    create_table :events do |t|
+      t.date :date, :null => false
+      t.string :location, :null => false
+      t.string :name, :null => false
```

Comment:
It's weird that Rubocop isn't picking this up but this is the old hash syntax.

`:null => false` becomes `null: false`